### ISSUE TITLE
How should we update the schema to add demographics and other attributes?

### ISSUE THREAD

**Issue by tillahoffmann** [shedding-hub/shedding-hub#5]
The current schema does not support demographics (such as sex or age) or other attributes (such as vaccination status or disease severity). There are (at least) two different versions to update the schema.

**Option 1**: Add attributes at the top level of each participant. This would look as follows.

```yaml
participants:
  - age: 38
    sex: female
    vaccinated: true
    measurements:
      - ...
```

**Option 2**: Add a nested attributes section. This would look as follows.

```yaml
participants:
  - attributes:
      age: 38
      sex: female
      vaccinated: true
    measurements:
      - ...
```

Option 1 is a bit shorter, but option 2 might be more explicit and machine-readable. What do you think, @WenfeiXiao, @YWAN446?

**Comment by WenfeiXiao** [comment]
I like the second option. It could potentially avoid any ordering issues and makes the structure more explicit and easier to parse.

**Comment by YWAN446** [comment]
A little late comment: I also like the Option 2!

**Comment by tillahoffmann** [comment]
Great, draft of the update schema is available in #7.
